### PR TITLE
[Blog] week 41 (2023)

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

File: weekly/2023/week-41.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - October
+---
+
+## Week 41 (October 10th – October 16th)
+
+- We have implemented several improvements in syncing the release downstream:
+  - Packit now updates ACL of its own dist-git forks to allow users and groups with commit rights to the original
+    dist-git repo to push directly to a fork. ([packit#2112](https://github.com/packit/packit/pull/2112))
+  - Packit now also properly cleans up the branch after syncing the release which should prevent unwanted files
+    (e.g.tarballs) being committed in dist-git. ([packit#2125](https://github.com/packit/packit/pull/2125))
```
Comment (mfocko) [inline weekly/2023/week-41.md:16]:
```suggestion
    (e.g. tarballs) being committed to dist-git. ([packit#2125](https://github.com/packit/packit/pull/2125))
```

File: weekly/2023/week-41.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+date: 2023-10-16
+tags:
+  - 2023-October
+  - 2023
+  - October
+---
+
+## Week 41 (October 10th – October 16th)
+
+- We have implemented several improvements in syncing the release downstream:
```
Comment (mfocko) [inline weekly/2023/week-41.md:12]:
```suggestion
- We have implemented several improvements in syncing the downstream release:
```

File: weekly/2023/week-41.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  - Packit now updates ACL of its own dist-git forks to allow users and groups with commit rights to the original
```
Comment (mfocko) [inline weekly/2023/week-41.md:13]: I still want to set Pagure ablaze for this…

File: weekly/2023/week-41.md (hunk 2 above)
Comment (lbarcziova) [reply weekly/2023/week-41.md:12]: I really wanted to say `sync the release downstream`, I can add `to`, but I think this should be correct as well

**Review by mfocko — APPROVED** [review]

**Pull request merged**